### DISCUSSION TITLE
Trouble creating a Regex to filter spam

### DISCUSSION THREAD

**Discussion by yontekh** [Chatterino/chatterino2#3343] [Q&A] [answered]
I would like to make a regex that filters out messages that
  - contain less than 3 words
     - a way to exclude words from this rule, like LOL or Kappa not counting against the 3 word minimum count

I just cannot figure it out... 
This could be added as a feature in the menu labeled "reduce spam" or something, but thats up to the devs.

**Comment by pajlada** [comment, marked as answer]
This would probably work OK as a plugin to filter messages - covered by this plugin feature request https://github.com/Chatterino/chatterino2/discussions/4999#discussioncomment-7746245
